server/balancer.go

```diff
@@ -370,3 +372,207 @@ func (r *replicaChecker) checkBestReplacement(region *RegionInfo) Operator {
 	}
 	return newTransferPeer(region, oldPeer, newPeer)
 }
+
+// RegionStateValue record each hot region state
+type RegionStateValue struct {
+	RegionID       uint64    `json:"region_id"`
+	WrittenBytes   uint64    `json:"written_bytes"`
+	UpdateCount    int       `json:"update_count"`
+	LastUpdateTime time.Time `json:"last_update_time"`
+	StoreID        uint64    `json:"-"`
+	antiCount      int
+	version        uint64
+}
+
+// MetaRegionStatus is a list of a group region state type
+type MetaRegionStatus []RegionStateValue
+
+func (m MetaRegionStatus) Len() int           { return len(m) }
+func (m MetaRegionStatus) Swap(i, j int)      { m[i], m[j] = m[j], m[i] }
+func (m MetaRegionStatus) Less(i, j int) bool { return m[i].WrittenBytes > m[j].WrittenBytes }
```

Comment:
I prefer `return m[i].WrittenBytes < m[j].WrittenBytes` here then use `sort.Sort(sort.Reverse(x))` to sort later. It will be semantically more accurate.